Calibration/HcalAlCaRecoProducers/plugins/AlCaIsolatedBunchFilter.cc

```diff
@@ -115,63 +110,54 @@ bool AlCaIsolatedBunchFilter::filter(edm::Event& iEvent,
   if ((trigIsoBunchNames_.size() == 0) && (trigJetNames_.size() == 0)) {
     accept = true;
   } else {
-    trigger::TriggerEvent triggerEvent;
-    edm::Handle<trigger::TriggerEvent> triggerEventHandle;
-    iEvent.getByToken(tok_trigEvt_, triggerEventHandle);
-    if (!triggerEventHandle.isValid()) {
-      edm::LogWarning("HcalIsoBunch") << "Error! Can't get the product "
-                                      << triggerEvent_.label() ;
-    } else {
-      triggerEvent = *(triggerEventHandle.product());
-      /////////////////////////////TriggerResults
-      edm::Handle<edm::TriggerResults> triggerResults;
-      iEvent.getByToken(tok_trigRes_, triggerResults);
-      if (triggerResults.isValid()) {
-        std::vector<std::string> modules;
-        const edm::TriggerNames & triggerNames = iEvent.triggerNames(*triggerResults);
-        const std::vector<std::string> & triggerNames_ = triggerNames.triggerNames();
-	bool jet(false), isobunch(false);
-        for (unsigned int iHLT=0; iHLT<triggerResults->size(); iHLT++) {
-          int hlt    = triggerResults->accept(iHLT);
-	  if (!jet) {
-	    for (unsigned int i=0; i<trigJetNames_.size(); ++i) {
-	      if (triggerNames_[iHLT].find(trigJetNames_[i].c_str()) !=
-		  std::string::npos) {
-		if (hlt > 0) jet = true;	
-		if (jet) {
+    /////////////////////////////TriggerResults
+    edm::Handle<edm::TriggerResults> triggerResults;
+    iEvent.getByToken(tok_trigRes_, triggerResults);
+    if (triggerResults.isValid()) {
+      std::vector<std::string> modules;
```

Comment:
@bsunanda still looks like this vector of strings `std::vector<std::string> modules` is not used anywhere